docs/test/starting_tests_fixtures.md

```diff
@@ -0,0 +1,49 @@
+# Run bdd tests in detached mode
+
+If you're interested to start the docker containers manually and run parts of the bdd tests on your own as opposed
+via the make target mentioned in [bddtests_instruction.md](bddtests_instruction.md), then do the following:
+
+1. Navigate to test/bdd/fixtures
+
+2. To start Alice and Bob agents and webhooks, go to agents folder and run:
+```shell script
+(source .env && docker-compose down && NO_PROXY=* docker-compose up --force-recreate)
+```
+   
+3. To start the Demo API hosts for Alic and Bob, open up a new terminal and go demo/openapi folder and run the 
+above command.
+
+4. To start SideTree Mock service, open up a new terminal go to sidetree-mock folder and run the same above command.
+
+5. Once all the containers above have started, you are ready to run BDD tests manually.
+
+6. In a new terminal, navigate to tests/bdd folder and run the same test command found in scripts/checke_integration.sh
+But you will need to add an env variable to instruct the tests to not start docker containers. The command would look
+like this:
+```shell script
+DISABLE_COMPOSITION=true go test -count=1 -v -cover . -p 1 -timeout=20m -race
+```
+The tests will start executing and you will notice logs written to the containers created above depicting 
+their activities.
+
+7. The above command will run all tests steps referenced in test/bdd/features. To run specific steps only, you can add 
+a run argument to specify the target steps you wish to run like the following:
+ ```shell script
+ DISABLE_COMPOSITION=true go test -run didexchange_e2e_sdk -count=1 -v -cover . -p 1 -timeout=20m -race
+ ```
+or
+ ```shell script
+ DISABLE_COMPOSITION=true go test -run all -count=1 -v -cover . -p 1 -timeout=20m -race
+ ``` 
+
+using `-run all` is the equivalent of not specifying the run argument.
+At the time of writing this doc, the following are available run targets, they are located in the .feature files:
```

Comment:
Looks like we are duplicating the content - https://github.com/hyperledger/aries-framework-go/blob/master/docs/test/bdd_tests.md#demonstration.

Reply:
this is the list of runnable features.. the demo is probably making use of these, but the list is not meant to be exhaustive as new feature will be added in the future (I doubt this readme will be updated then :))

Reply:
yeah, but may be pointing to one instance(either this or on the page shared on first link) would be helpful. Duplicates are hard to maintain and quickly goes out of sync.

Reply:
the intent was not to maintain the list.. just to show examples.. and the reader can figure out other features by reading the files.

Reply:
removed list in the new doc now.